cmd/cronosd/config/config.go

```diff
@@ -2,6 +2,11 @@ package config
 
 import sdk "github.com/cosmos/cosmos-sdk/types"
 
+const (
+	NetworkMainet = iota + 1
+	NetworkTestnet
```

Comment:
Maybe consider `TestnetPoseidon` or `Testnet<XXX>`? if we will have more than 1 testnet in the future?

Reply:
yeah, I guess it's not a good idea to do the check here, we don't know about different testnet networks at compile time, so I added a different plan name for testnet3 to upgrade, `v1.0.0-testnet3`.